components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -114,7 +116,7 @@ be temporary, and you can try again later.</li>
 <p>Though the site seems valid, the browser was unable to establish a connection.</p>
 <ul>
 <li>Could the site be temporarily unavailable? Try again later.</li>
-<li>Are you unable to browse other sites?  Check the computer\'s network connection.</li>
+<li>Are you unable to browse other sites?  Check the computer\’s network connection.</li>
```

Comment:
you can remove the backslash (previously for escaping the `'`)  now, correct?

Reply:
I second that :)

Reply:
can we remove the extra space after the `?` here too?